AEPCore/Sources/configuration/LaunchIDManager.swift

```diff
@@ -39,6 +39,7 @@ struct LaunchIDManager {
     /// - Returns: appId loaded from persistence, nil if not present
     func loadAppIdFromPersistence() -> String? {
         if let appId = dataStore.getString(key: ConfigurationConstants.DataStoreKeys.PERSISTED_APPID) {
+            Log.trace(label: logTag, "Loading App ID from persistence")
```

Comment:
Can you also add appId to log message? Even better option would be to move this log message to calling function.

Reply:
I agree that a better option would be to remove logs from these functions and move to calling function but what do you think about the following: We call `loadAppIdFromPersistence` and `loadAppIdFromManifest` from the `loadAppId` function. 

```
    func loadAppId() -> String? {
        // Prefer appId stored in persistence over in manifest
        return loadAppIdFromPersistence() ?? loadAppIdFromManifest()
    }
```

So we either change how clean this function is in order to provide logs here. Or we provide logs in the callers of `loadAppId()` but then we wont get specific logs knowing if the app id was loaded from manifest or persistence. Just that it was loaded.

Reply:
I am fine with the current logs. Can we change the current logs to also print App Id as it will be useful for debugging.            
`Log.trace(label: logTag, "Loaded App ID \(appId) from persistence")`